### DISCUSSION TITLE
adding a DatabaseCluster datasource AppSync via the CDK (go)

### DISCUSSION THREAD

**Discussion by crispywalrus** [aws/aws-cdk#29733] [Q&A]
I have an `awsappsync.GraphqlApi` and a `awsrds.DatabaseCluster` The ability to enable the Data API was recently added to the CDK, so I've got that. In order to add a resolver I need a datasource to add it to and  I can't figure out how to create a datasource from the cluster that's acceptable. Type wise the `api.AddRdsDataSource` requires an `IServerlessCluster` an interface that `DatabaseCluster` doesn't implement. Do I use AddHttpDatasource (that seems wrong) what am I missing?

**Comment by ghost** [comment]
I want to ensure aws isn't blocking or restricting my incoming and out going messages. Please assist
